app/config.go

```diff
@@ -29,21 +28,15 @@ const (
 )
 
 func (s *Server) Config() *model.Config {
-	if cfg := s.config.Load(); cfg != nil {
-		return cfg.(*model.Config)
-	}
-	return &model.Config{}
+	return s.configStore.Get()
```

Comment:
I've been thinking about making `Get()` do a clone of the config, so that it's never possible to modify the actual config without invoking `Set` or `Patch`. We call `Config()` a /lot/, so there /might/ be some noticeable overhead to doing this, at least without a more efficient `Clone`. Either way, this breaks stuff so I'm deferring that as noted in the summary.